scalafmt-tests/src/test/resources/unit/Source3.source

```diff
@@ -0,0 +1,12 @@
+<<< kind projector #2560 
+runner.dialect = scala213source3
+===
+class x(arg: OptionT[Either[Int, +_]])
```

Comment:
if you wrap in `object a { ... }` or `package a { ... }`, you can use an existing `.stat` file.

Reply:
Done